celo-org/celo-monorepo
# [wallet]Run geth in an infura-like mode

### Description

Run mobile app without running a local geth instance. Connect to `{testnet}-infura.celo-testnet.org` for example, https://integration-infura.celo-testnet.org`

### Tested

1. Set `DEFAULT_TESTNET=integration` and `DEFAULT_SYNC_MODE=-1` in `packages/mobile/.env` file. 
2. Run the app with `yarn run dev`.
3. Invite yourself with `celotooljs account invite -e integration --phone <phone number>` to get SMS.
4. Perform the verification, send money, exchange gold, ensure that all of this works.

### Related issues

- Fixes #342
